### PR TITLE
Added support for MongoDB Export of results for Instruction/Prompt Template

### PR DESCRIPTION
**Pull request by pramitbhatia25**
Summary of changes:
Added export to_mongo_db option after generating results.
Added error handling in experiment.py to_mongo_db function.
Modified open_ai_chat_completion and open_ai_chat experiments, made them set completion functions only if DEBUG key doesn't exist.
Modified load_data function in data_loader.py to return a dict. If there's any error, it is displayed on the UI instead of the app crashing.
Added Anthropic / Google PALM / LlamaCpp Completion models in data loader so they don't error out when called.
(Support for these still needs work)

Important: Since I've modified data_loader.py / experiment.py, and the playground imports these from the prompttools package, we'll need to release a new version and update it. (I've tested the functions by using them directly in playground.py, they were working fine..)

### CODE REVIEW

**Comment by pramitbhatia25** [comment]
Hey guys. let me know if I should make any more modifications. I'm working on the to_mongo_db connection for Model Comparison, will need to modify the run_multiple function in data_loader.py file to get it working.

File: prompttools/playground/data_loader.py [hunk 1]
```diff
@@ -35,8 +35,10 @@ def load_data(
     presence_penalty=0.0,
     api_key=None,
 ):
+    data_loader_response = {'error': []}
```
Comment (steventkrawczyk) [inline prompttools/playground/data_loader.py:38]: For error handling, why not just raise an error? Streamlit will show that to the user

File: requirements.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@ ipywidgets
 sentence-transformers
 pylint
 streamlit
+pymongo
```
Comment (steventkrawczyk) [inline requirements.txt:11]: Since we guard the import, we don't need to install here.

**Comment by steventkrawczyk** [comment]
This is a great start! 
* You'll need to rebase on the latest changes from today, we're updating the UI a bit
* Could you attach a few screenshots to show how the UI will look with these changes?

**Review by steventkrawczyk — CHANGES_REQUESTED** [review]
Looks good so far! Left a few comments

**Comment by steventkrawczyk** [comment]
One more idea here: can we use streamlit's connectors for mongodb? 

https://docs.streamlit.io/knowledge-base/tutorials/databases/mongodb

**Comment by pramitbhatia25** [comment]
Gotcha, I'll make these changes, will open another PR when I'm done. Thanks!

**Pull request closed without merging**